curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa2aec2f09d454253aad6c.md

```diff
@@ -0,0 +1,394 @@
+---
+id: 62aa2aec2f09d454253aad6c
+title: Step 171
+challengeType: 0
+dashedName: step-171
+---
+
+# --description--
+
+Since you subtracted health from the player, you need to check `if` the player's `health` is less than `0`. If it is, call the `lose` function.
```

Comment:
```suggestion
Since you subtracted health from the player, you need to check `if` the player's `health` is less than or equal to `0`. If it is, call the `lose` function.
```